components/sensor/mpr121.rst

```diff
@@ -0,0 +1,60 @@
+MPR121 Capacitive Touch Sensor
+===========================================
+
+.. seo::
+    :description: Instructions for setting up MPR121 Capacitive Touch Sensor
+    :image: mpr121_breakout.jpg
+    :keywords: MPR121
+
+The ``mpr121`` sensor platform allows you to use your MPR121
+(`datasheet <https://cdn-learn.adafruit.com/downloads/pdf/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial.pdf>`__,
+`Adafruit`_) Capacitive Touch Sensor with esphomelib. The :ref:`I²C <i2c>` is
+required to be set up in your configuration for this sensor to work.
+
+.. figure:: images/mpr121_keypad.jpg
+    :align: center
+    :width: 20.0%
+
+    MPR121 Capacitive Touch Sensor.
+
+.. _Adafruit: https://learn.adafruit.com/adafruit-mpr121-12-key-capacitive-touch-sensor-breakout-tutorial/overview
+
+.. code-block:: yaml
+
+    # Example configuration entry
+	mpr121:
```

Comment:
This indentation appears to be faulty, mpr121: should have the same indentation as the comment above